homeassistant/components/esphome/light.py

```diff
@@ -388,3 +411,10 @@ def _on_static_info_update(self, static_info: EntityInfo) -> None:
         self._attr_effect_list = static_info.effects
         self._attr_min_mireds = round(static_info.min_mireds)
         self._attr_max_mireds = round(static_info.max_mireds)
+        self._attr_min_color_temp_kelvin = _mired_to_kelvin(static_info.max_mireds)
+        self._attr_max_color_temp_kelvin = _mired_to_kelvin(static_info.min_mireds)
+
+    @property
```

Comment:
Is this leftovers from the merge conflict?

Reply:
No, this is intentional. By adding the `_attr_{min,max}_color_temp_kelvin` class properties here, we don't need to implement the base class `{min,max}_color_temp_kelvin()` functions, which simplifies the proposed code quite a bit. I tested with and without these properties; with the properties it works as expected. Without, the upper/lower bounds of the color temperature selector become inaccurate.

Reply:
I was referring to the `effect_list` property 
```
    def effect_list(self) -> list[str]:
        """Return the list of supported effects."""
        return self._static_info.effects
```